engines/bastion_katello/app/assets/javascripts/bastion_katello/i18n/bastion_katello.pot

```diff
@@ -2406,8 +2406,7 @@ msgstr ""
 
 #: app/assets/javascripts/bastion_katello/content-views/details/views/content-view-publish.html
 msgid ""
-"For the duplicate repositories, the packages in each Content View Version will combine regardless of the filters that \n"
-"        were applied to the individual Content View Version."
+"Duplicate repositories between the selected Content View Versions will merge, resulting in a Composite Content View with all packages that exist among the duplicates."
```

Comment:
these files aren't meant to be updated manually

Reply:
Figured out how to generate them.

Reply:
oh no, i mean they aren't supposed to be updated at all as part of a pr.  All translation strings are updated around release time.  Apologies if i wasn't clear

Reply:
Ah I gotcha, I'll remove it then